README.md

```diff
@@ -33,6 +33,17 @@ Run:
 ```
 All files you add/create into your forked version will be always safe from being overriden when you get latest from upstream (qgds-tokens).
 
+### What files should I look for or change when creating your custom theme
+Please add a reference of your new theme on $metadata.json and on $themes.json. Please see the example branch for more info: custom-tokens-and-themes-example
```

Comment:
should make it a link to the comparison.
Since that work is not pushed yet its still 'pending'.

https://github.com/qld-gov-au/qgds-tokens/compare/v3.1.5...custom-tokens-and-themes-example